### PR TITLE
add style

### PR DESCRIPTION
**Pull request by Agetuni**
- style the whole page
- 
image

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -2,16 +2,26 @@ import './header.css';
...
+        </li>
+        <li>
+          <Link to="/catagory" className="nav-category">
+            Category
+          </Link>
+        </li>
+      </ul>
+    </nav>
+    <div>
+      <div className="ovalheader" />
```
Comment (fobadara) [inline src/components/header.js]: [REQUIRED] The round avatar icon on the right f the template header is not present in yours

File: src/components/header.js [hunk 2]
```diff
@@ -2,16 +2,26 @@ import './header.css';
...
+      <ul>
+        <li>
+          <Link to="/" className="nav-books">
+            {' '}
+            Books
+          </Link>
+        </li>
+        <li>
+          <Link to="/catagory" className="nav-category">
+            Category
```
Comment (fobadara) [inline src/components/header.js:18]: [REQUIRED] Kindly make this lighter and remove the underline just as it is in the template.

File: src/components/header.js [hunk 3]
```diff
@@ -2,16 +2,26 @@ import './header.css';
...
-    </ul>
-  </nav>
+  <div className="header">
+    <nav className="panel-bg">
+      <span className="nav-logo">Bookstore CMS</span>
+      <ul>
+        <li>
+          <Link to="/" className="nav-books">
+            {' '}
+            Books
```
Comment (fobadara) [inline src/components/header.js:13]: [REQUIRED] Kindly remove the underline

File: src/components/book.js [hunk 4]
```diff
@@ -14,32 +14,51 @@ const Book = (props) => {
...
+            >
+              Remove
+            </button>
           </li>
           <li>
             {' '}
-            <button type="button">edit</button>
+            <button className="buttons" type="button">edit</button>
           </li>
         </ul>
```
Comment (fobadara) [inline src/components/book.js:48]: [REQUIRED] Kindly make the dark line between comments, edit and remove more faint.

[REQUIRED] kindy ensure that the button container is well aligned on the left just like the author and title above it.

|Your Design|
|------------|
|image|

File: src/components/book.js [hunk 5]
```diff
@@ -14,32 +14,51 @@ const Book = (props) => {
...
+        <div className="oval" />
+        <div className="completed">
+          <span className="percent-value">65%</span>
+          <span className="percent-compeleted">Completed</span>
+        </div>
+      </div>
+      <div className="progress">
+        <span className="currentChapter">CURRENT CHAPTER</span>
+        <span className="chapter">Chapter 17</span>
+        <button type="button" className="update"> UPDATE PROGRESS</button>
```
Comment (fobadara) [inline src/components/book.js:60]: [REQUIRED] Kindy remove the dark border on all `update progress buttons`

File: src/components/addbook.css [hunk 6]
```diff
@@ -4,7 +4,8 @@
 
 .addbook-form {
   width: 86rem;
-  margin: 2.313rem 6.25rem 0;
+  margin: 2.313rem 6.25rem 2rem;
+  padding-bottom: 100px;
 }
```
Comment (fobadara) [inline src/components/addbook.css]: [REQUIRED] Kindly implement the requirements in the image below:

| Template |
|----------|
|image|

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Agetuni, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._
I had to include some comments here because I can't access some files for inline comment

[REQUIRED] Kindly fix the issue with the overflow. Your entire page is overflowing to the left and right

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Agetuni** [comment]
image

**Review by Munsa1 — APPROVED** [review]
Hi @Agetuni, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**